Userland/Games/Snake/Game.h

```diff
@@ -14,9 +14,11 @@
 namespace Snake {
 
 class Game : public GUI::Frame {
-    C_OBJECT(Game);
+    C_OBJECT_ABSTRACT(Game);
```

Comment:
What exactly makes this widget suddenly be virtual?

Reply:
It's already virtual, but isn't what this does. This just let's subclasses define their own `try_create`.

Reply:
Yeah it's just a bad name. We had a discussion about using that macro like this months ago, and it's not ideal but it's the only way currently of letting you define the factory functions manually.